specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2024-03-01/amlfilesystem.json

```diff
@@ -0,0 +1,1453 @@
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "AmlFilesystemArchiveInfo": {
+      "type": "object",
+      "description": "Information required to execute the archive operation",
+      "properties": {
+        "filesystemPath": {
+          "type": "string",
+          "description": "Lustre file system path to archive relative to the file system root.  Specify '/' to archive all modified data.",
+          "default": "/"
+        }
+      }
+    },
+    "AmlFilesystemArchive": {
+      "description": "Information about the AML file system archive",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "filesystemPath": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Lustre file system path to archive relative to the file system root.  Specify '/' to archive all modified data."
+        },
+        "status": {
+          "description": "The status of the archive",
+          "type": "object",
+          "readOnly": true,
+          "properties": {
+            "state": {
+              "readOnly": true,
+              "description": "The state of the archive operation",
+              "type": "string",
+              "enum": [
+                "NotConfigured",
+                "Idle",
+                "InProgress",
+                "Canceled",
+                "Completed",
+                "Failed",
+                "Cancelling",
+                "FSScanInProgress"
+              ],
+              "x-ms-enum": {
+                "name": "ArchiveStatusType",
+                "modelAsString": true
+              }
+            },
+            "lastCompletionTime": {
+              "readOnly": true,
+              "description": "The time of the last completed archive operation",
+              "type": "string",
+              "format": "date-time"
+            },
+            "lastStartedTime": {
+              "readOnly": true,
+              "description": "The time the latest archive operation started",
+              "type": "string",
+              "format": "date-time"
+            },
+            "percentComplete": {
+              "readOnly": true,
+              "description": "The completion percentage of the archive operation",
+              "type": "integer",
+              "format": "int32"
+            },
+            "errorCode": {
+              "readOnly": true,
+              "description": "Server-defined error code for the archive operation",
+              "type": "string"
+            },
+            "errorMessage": {
+              "readOnly": true,
+              "description": "Server-defined error message for the archive operation",
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "AmlFilesystemImportInfo": {
+      "type": "object",
+      "description": "Information required to execute the import operation",
+      "properties": {
+        "importId": {
```

Comment:
>importId [](http://example.com/codeflow?start=9&length=8)

what is this importid ? does it refer to some other well known id ? Can you also update the description ?

Reply:
@raosuhas answered by Brian above. It serves as a key for the import for a future case where we may have than one